pkgs/servers/matrix-conduit/default.nix

```
@@ -1,22 +1,40 @@
-{ stdenv, lib, fetchFromGitLab, rustPlatform, }:
+{ stdenv, lib, fetchFromGitLab, rustPlatform, pkgs, ... }:
```

```suggestion
{ stdenv, lib, fetchFromGitLab, rustPlatform, pkgs }:
```

Are the `…` needed for a specific reason?

---

No reason. Fixed.